data/filters/wg21.py

```diff
@@ -259,8 +260,11 @@ def header(elem, doc):
     if elem.identifier == 'bibliography':
         elem.classes.remove('unnumbered')
 
-    elem.content.append(
-        pf.Link(url=f'#{elem.identifier}', classes=['self-link']))
+    string_url = f'#{elem.identifier}'
+    self_link = pf.Link(url=string_url, classes=['self-link'])
+    self_links[string_url] = pf.stringify(elem)
+
+    elem.content.append(self_link)
```

Comment:
No need to pull out `self_link` here.
```suggestion
    elem.content.append(pf.Link(url=url, classes=['self-link']))
```

Reply:
Done